src/IPriceCalculator.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+interface IPriceCalculator {
```

Comment:
Having the price calculation being done on a separate contract means that if in the future we want to move away from linear calculation to a curve, it should be possible by just changing the current `priceCalculator` used by the `Membership` contract